include/alpaka/core/ConcurrentExecPool.hpp

```diff
@@ -367,7 +385,8 @@ namespace alpaka::core
             std::vector<TConcurrentExec> m_vConcurrentExecs;
             ThreadSafeQueue<std::shared_ptr<ITaskPkg>> m_qTasks;
             std::atomic<std::uint32_t> m_numActiveTasks;
-            std::atomic<bool> m_bShutdownFlag;
+            std::atomic<bool> m_bShutdownFlag, m_bDetachedFlag = false;
```

Comment:
If NSDMI works the same way variable initialization works in other places, then this does not do what you think.
Only `m_bDetachedFlag` is set to `false` and `m_bShutdownFlag` is default initialized.
Please split this initialization in multiple lines.

Reply:
It does work the same way and intended to get this behaviour. So, the two-line version would be:
```c++
std::atomic<bool> m_bShutdownFlag;
std::atomic<bool> m_bDetachedFlag = false;
```
`m_bShutdownFlag` was there before and is initialized in the c'tor. I did not want to touch code I do not need to change but also did not want to use old-style initialization for the new member.

I can see how this inconsistency obfuscates the intent, so I guess it is best to update the trivial initializations of the pre-existing members.